Formatting Pass

davidkwoods opened the pull request:
- Performed formatting pass across C# solutions
- Updated Gradle setup to allow Java build to succeed
- Added .gradle/ to .gitignore

#55403
#55017

davidkwoods committed:
Getting Gradle to Build

- Setting InvalidPackage to fail as a warning, since it can be [unreliable](https://stackoverflow.com/a/37914534)
- Adding .gradle/ to .gitignore

#55403
#55017

msftclas commented:
@davidkwoods,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

davidkwoods commented:
FYI, I'm amending these changes.  It seems the "Remove and Sort Usings" step was overly aggressive so I need to go through and add them back to make sure things build.

davidkwoods committed:
Formatting Pass

- Ran CoreFX CodeFormatter on solutions
- Ran Visual Studio's Format Document on solution files
- Manual smoke check to correct anomalies

#55403

RandomEngy commented:
:shipit:

davidkwoods commented:
Sandbox build (more complete than the PR, and more reliable so far) succeeded, so I'm going to move ahead with this PR.
